hotfix_process.md

```diff
@@ -0,0 +1,174 @@
+      RUN dnf update -y
+      RUN dnf install -y https://cbs.centos.org/kojifiles/packages/python-django-pyscss/2.0.3/1.el9s/noarch/python3-django-pyscss-2.0.3-1.el9s.noarch.rpm
+    binary:
+      asFile: ""
+  strategy:
+    type: Docker
+    dockerStrategy:
+      env:
+        - name: NODE_ENV
+          value: production
+  triggers:
+    - type: ConfigChange
+```
+Apply the manifest using the openshift client tool:
+
+```bash
+$ oc apply horizon-hotfix.yaml
+```
+
+Run the following command to confirm that the hotfix image has been
+successfully pushed to the OpenShift internal registry:
+
+```bash
+$ oc get imagestream -n openstack
+
+NAME             IMAGE REPOSITORY                                                                   TAGS     UPDATED
+horizon-hotfix   default-route-openshift-image-registry.apps-crc.testing/openstack/horizon-hotfix   latest   26 seconds ago
+```
+
+To apply the new image, update the `OpenStackVersion` custom resource and
+include the hotfix image under `spec.customContainerImage`. For example:
+
+```yaml
+spec:
+  customContainerImage:
+    horizonImage: image-registry.openshift-image-registry.svc:5000/openstack/horizon-hotfix:latest
+```
+For more information about the `OpenStackVersion` approach see [OpenStackVersion custom-images
+document](version_updates.md#custom-images-for-other-openstack-services).
+
+**Note:**
+Replace the image path with your actual internal registry route if different.
+
+
+### Expose the openshift-image-registry
+
+By default, the OpenShift internal image registry (`openshift-image-registry`) is
+not externally accessible. However, exposing it can be useful when non-cluster
+components—such as `EDPM` nodes need to pull patched container images directly
+(e.g., hotfixes built with OpenShift `BuildConfig` described in the previos
+section).
+
+This section explains how to expose the internal registry using a secure
+`Route`, verify access, and pull images using `podman`.
+
+1. Ensure the `openshift-image-registry` project exists and that the registry pods
+and services are running:
+
+```bash
+$ oc get pods -n openshift-image-registry
+NAME                                               READY   STATUS      RESTARTS   AGE
+cluster-image-registry-operator-7769bd8d7d-4xwcp   1/1     Running     2          13h
+image-registry-56f7846646-55tj4                    1/1     Running     0          11h
+node-ca-98n8x                                      1/1     Running     1          12h
+node-ca-jr5qn                                      1/1     Running     1          12h
+node-ca-x24kt                                      1/1     Running     1          12h
+
+$ oc get svc -n openshift-image-registry
+NAME                      TYPE        CLUSTER-IP     EXTERNAL-IP   PORT(S)     AGE
+image-registry            ClusterIP   172.30.80.87   <none>        5000/TCP    12h
+image-registry-operator   ClusterIP   None           <none>        60000/TCP   13h
+```
```

Comment:
Maybe could mention this and its output here too:

```
$ oc get clusteroperator image-registry
NAME             VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE   MESSAGE
image-registry   4.18.4    True        False         False      8d   
```